test/functionalTest/cases/0950_GET_v2_entity/GET_v2_entity_too_many.test

```diff
@@ -0,0 +1,159 @@
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+GET Entity V2 To Many Entities
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. POST /v1/updateContext, creating 5 entities with id E1
+# 02. GET /v2/entities/E1 (too many entities error)
+#
+
+echo "01. POST /v1/updateContext, creating 5 entities with id E1"
+echo "=========================================================="
+payload='{
+  "contextElements": [
+    {
+      "id":   "E1",
+      "type": "T1"
+    },
+    {
+      "id":   "E1",
+      "type": "T2"
+    },
+    {
+      "id":   "E1",
+      "type": "T3"
+    },
+    {
+      "id":   "E1",
+      "type": "T4"
+    },
+    {
+      "id":   "E1",
+      "type": "T5"
+    }
+  ],
+  "updateAction": "APPEND"
+}'
+orionCurl --url /v1/updateContext --payload "$payload" --json
+echo
+echo
+
+echo "02. GET /v2/entities/E1 (too many entities error)"
+echo "================================================="
+orionCurl --url '/v2/entities/E1' --json
+echo
+echo
+
+--REGEXPECT--
+01. POST /v1/updateContext, creating 5 entities with id E1
+==========================================================
+HTTP/1.1 200 OK
+Content-Length: 1072
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T1"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T2"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T3"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T4"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        },
+        {
+            "contextElement": {
+                "id": "E1",
+                "isPattern": "false",
+                "type": "T5"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        }
+    ]
+}
+
+
+02. GET /v2/entities/E1 (too many entities error)
+=================================================
+HTTP/1.1 409 Conflict
+Content-Length: 73
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "description": "/v2/entities?id=E1",
```

Comment:
"Refine your query by providing a type" could be missleading: adding a type couldn't solve the problem if "duplicity" is due to service path. Thus, let's keep it with "Theres is more than one entity with id X. Refine your query" (without mentionint type, as it is only one way of refining, but not the only one and not necessarily the one that would provide the solution).